### PR TITLE
install/completion: add tabs, add tcsh

### PR DESCRIPTION
**Pull request by casperdcl**
- [x] add nested tabs [because they're neater](https://github.com/iterative/dvc.org/pull/4123#issuecomment-1322804452) :)
- [x] add `tcsh`
- [x] fix/work-around [rendering overflow](https://github.com/iterative/dvc.org/pull/4123#pullrequestreview-1189056725)
- [x] improve explanations
- [x] rebase on `main`/fix conflicts

### CODE REVIEW

File: redirects-list.json [hunk 1]
```diff
@@ -38,6 +38,9 @@
   "^/features$                                                                            /doc/use-cases 302",
 
   "^/doc/install/plugins(/.*)?$                                                           /doc/install/ide-plugins$1",
+  "^/doc/install/completion#bash-completion-on-macos$                                     /doc/install/completion?tab=Bash-macOS",
+  "^/doc/install/completion#bash-completion-on-debianubuntu$                              /doc/install/completion?tab=Bash-Debian-Ubuntu",
+  "^/doc/install/completion#zsh-completion$                                               /doc/install/completion?tab=Zsh",
```
Comment (jorgeorpinel) [inline redirects-list.json:43]: I don't think that the URL anchors make it to the router. Can you confirm @iterative/websites ? Thanks

Comment (jorgeorpinel) [reply redirects-list.json:43]: In any case we don't have any links to these sections directly from our docs so I wouldn't bother with these.

**Review by jorgeorpinel — DISMISSED** [review]
Unfortunately nested tabs don't render neatly [1]. Is there a non-nested alternative that still uses tabs? What are the pros? A con is losing the CONTENT links [2].

1
image

> https://dvc-org-completion-tabs-fmuzor.herokuapp.com/doc/install/completion?tab=Bash-macOS

2
image

> https://dvc.org/doc/install/completion

**Comment by casperdcl** [comment]
A "pro" is much less scrolling required, so "content" links aren't even needed.

File: redirects-list.json (hunk 1 above)
Comment (casperdcl) [reply redirects-list.json:43]:
```suggestion
  "^/doc/install/plugins(/.*)?$                                                           /doc/install/ide-plugins$1",
```

**Review by jorgeorpinel — APPROVED** [review]
Let's not merge before https://github.com/iterative/gatsby-theme-iterative/issues/138 though.

**Comment by casperdcl** [comment]
Found a [different work-around](https://github.com/iterative/dvc.org/pull/4123/commits/aaf0600e0f1cd9a57d4cbaa73d4b0b133ceb80e3), happy to merge now?

I'd like it merged so I can link to these improved docs when chasing up some external issues ;)

**Comment by shcheklein** [comment]
I like the approach!

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
Looks good, thanks for the workaround.

**Pull request merged**